frontends/web/src/routes/buy/exchange.tsx

```diff
@@ -86,9 +91,23 @@ export const Exchange = ({ code, accounts }: TProps) => {
     }
     const regionNames = new Intl.DisplayNames([locale], { type: 'region' });
     const regions = regionList.regions.map(region => ({ value: region.code, text: regionNames.of(region.code) } as TOption));
+
     regions.sort((a, b) => a.text.localeCompare(b.text, locale));
     setRegions(regions);
-  }, [regionList, locale]);
+
+    if (config && config.frontend.selectedExchangeRegion) {
```

Comment:
maybe add a check on `config` and `nativeLocale` at the beginning, to avoid multiple executions?